DeepFried2/layers/BatchNormalization.py

```diff
@@ -32,8 +32,8 @@ def symb_forward(self, symb_input):
             d_shuffle += ('x', 'x')
             axis += (2, 3)
         elif symb_input.ndim == 5:
-            d_shuffle = ('x', 'x', 0, 'x', 'x')
-            axis = (0, 1, 3, 4)
+            d_shuffle = ('x', 0, 'x', 'x', 'x')
+            axis = (0, 2, 3, 4)
```

Comment:
These could now all be merged, but I think that'd obfuscate the code.
Still, could you make the `5` case analogous to the 4 case? (i.e. use +=)